packages/client/hmi-client/src/services/model-configurations.ts

```diff
@@ -67,3 +74,75 @@ export const updateModelConfiguration = async (config: ModelConfiguration) => {
 	const response = await API.put(`/model-configurations/${config.id}`, config);
 	return response?.data ?? null;
 };
+
+export const getInitial = (config: ModelConfiguration, initialId: string): Initial | undefined =>
+	config.configuration.semantics?.ode.initials?.find((initial) => initial.target === initialId);
+
+export const getInitialSource = (config: ModelConfiguration, initialId: string): string =>
+	config.configuration.metadata?.initials?.[initialId].source ?? '';
+
+export const setInitialSource = (
+	config: ModelConfiguration,
+	initialId: string,
+	source: string
+): void => {
+	const initial = config.configuration.metadata?.initials?.[initialId];
+	if (initial) {
+		initial.source = source;
+	}
+};
+
+export const getParameter = (
+	config: ModelConfiguration,
+	parameterId: string
+): ModelParameter | undefined =>
+	config.configuration.semantics?.ode.parameters?.find((param) => param.id === parameterId);
+
+export const setDistribution = (
+	config: ModelConfiguration,
+	distribution: ModelDistribution,
+	parameterId: string
+): void => {
```

Comment:
In other functions you have the pattern of `object => key => value`, but here you have `object => value => key` ... can we make this more consistent?

Reply:
can do